package.json

```diff
@@ -2,104 +2,98 @@
   "name": "nativefier",
   "version": "7.7.1",
   "description": "Wrap web apps natively",
+  "license": "MIT",
+  "author": "Goh Jia Hao",
+  "engines": {
+    "node": ">= 8.10.0",
```

Comment:
Maybe bumping this to 10 would make sense? Node.js 8 seems to be end-of-life and oldest LTS supported is 10, with 12 being the active one: https://nodejs.org/en/about/releases/

Reply:
> Maybe bumping this to 10 would make sense? Node.js 8 seems to be end-of-life and oldest LTS supported is 10, with 12 being the active one: [nodejs.org/en/about/releases](https://nodejs.org/en/about/releases/)

@remusao it's too early to drop 8, all our dependencies support it, it's what ships in Debian stable, it's not excruciatingly old, I'm fine keeping it, the only very minor cost is a worse compilation target for TS, so slightly uglier js. Not worth alienating users happily running Node 8 for good reasons.